### PR TITLE
fix: blurry svgs in safari

### PR DESCRIPTION
**Pull request by LeahMarieBush**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-leah-fixblurry-svgs-safari-hashicorp.vercel.app/hcp) 🔎
- [Asana task](https://app.asana.com/0/1206815131022336/1203910559645521/f) 🎟️

## 🗒️ What

This PR updates the `try-hcp-callout` side image to use the svg inline vs as the background image. This is due to a bug in safari where the svgs look blurry. This PR fixes that issue and also another issue where svg ids are being duplicated (more context [here](https://github.com/hashicorp/web-platform-packages/pull/268))

## 📸 Design Screenshots

Before
Screenshot 2024-07-09 at 3 18 37 PM

After
Screenshot 2024-07-09 at 3 18 20 PM

## 🧪 Testing

1. Go to safari with [the preview link](https://dev-portal-git-leah-fixblurry-svgs-safari-hashicorp.vercel.app/hcp)
2. Look at the try-hcp-callout component on the page
3. Note that it is no longer blurry.
4. Also go to other product landing pages and make sure those images aren't blurry as well.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203910559645521

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
 		"@datadog/datadog-api-client": "^1.0.0-beta.7",
 		"@hashicorp/platform-cli": "^2.8.0",
 		"@hashicorp/platform-configs": "^0.2.0",
-		"@hashicorp/platform-nextjs-plugin": "^5.1.0",
+		"@hashicorp/platform-nextjs-plugin": "^5.1.1",
```
Comment (nandereck) [inline package.json:51]: Is this change related to the blurry SVG issue? Or is it separate? 

Also, I'm noticing that [rolling back this version](https://github.com/hashicorp/dev-portal/pull/2510) seems to result in the same bundle size.

File: src/components/try-hcp-callout/content.ts [hunk 2]
```diff
@@ -5,10 +5,17 @@
 
 import { TryHcpCalloutProps } from 'components/try-hcp-callout/types'
 import { ProductSlugWithContent } from './types'
+import boundarySvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-boundary.svg?include'
+import consulSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-consul.svg?include'
+import packerSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-packer.svg?include'
+import terraformSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-terraform.svg?include'
+import vagrantSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-vagrant.svg?include'
+import vaultSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-vault.svg?include'
+import waypointSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-waypoint.svg?include'
```
Comment (nandereck) [inline src/components/try-hcp-callout/content.ts:14]: If the increase in bundle size is in fact due to this change, I wonder if either there are [some optimizations](https://jakearchibald.github.io/svgomg/) that can be made to the SVG or if we can use a different file format.

**Review by nandereck — COMMENTED** [review]
This appears to address the issue but have some clarification questions related to the package update and the bundle size!

File: package.json (hunk 1 above)
Comment (LeahMarieBush) [reply package.json:51]: Yes when I used the inline svg for the images, the svg on the hcp page was cut off. That was because the product icon svg had a clip-path id of a and this svg had a clip-path id of a as well. I had to adjust our svg loader in the nextjs-plugin to address this. There is more info in the PR: https://github.com/hashicorp/web-platform-packages/pull/268

Also for bundle size, I assumed it was from this addition but it looks like it was not. I'll take a look at the bundle size today and see if I can decrease it

**Review by nandereck — APPROVED** [review]
LGTM! 🥇 Awesome work!

Mind removing the note about the bundle size in your PR description before merging?
